wishManagement/wish.css

```diff
@@ -0,0 +1,180 @@
+@import url("https://infusify.netlify.app/styles/Infusify.css");
+
+.flex-container{
+    display:flex;
+}
+.flex-rows{
+    flex-direction: row;
+}
+
+.flex-columns{
+    flex-direction: column;
+}
+
+.nav-vertical{
+    grid-area: aside;
+    background-color: var(--container-border-white);
+    /* height: calc(100vh - 9%); */
+    height:100vh;
+    padding:2rem;
+    width:max-content;
+    position: sticky;
+    top:0px;
+    bottom:100px;
+    overflow-y: scroll;
+}
+
+.nav-vertical .btn-link{
+    background-color: transparent;
+    font-size:1.2rem;
+    color:var(--hot-pink);
+    padding:0;
+    padding-top: 0.4rem;
+    margin-left:3rem;
+}
+
+.mb{
+    margin-bottom:0.8rem;
+}
+
+.ml{
+    margin-left:4rem;
+}
+
+.mt{
+    margin-top:1rem;
+}
+
+.text-small{
+    font-size: 1rem;
+    margin-left: 0.3rem;
+}
+
+.brand{
+    text-decoration: none;
+}
+
+.card-subtitle{
+    margin-bottom: 0.3rem;
+    font-size: 1rem;
+}
+
+.card-shadow{
+    height:380px;
+    width:80%;
+}
+
+#card-container{
+    background-color: transparent;
+}
+
+.wish-add{
+    background-color: var(--light-pink) !important;
+    color:var(--violet) !important;
+    border:none !important;
```

Comment:
Suggestion: don't use !important wherever possible

Reply:
Thank you for pointing it out, removed it